monero-outreach-docs/translations/es/PSA/files/teamwork/teamwork_es.sla

```diff
@@ -64,15 +67,44 @@
-            <trail LINESP="13.2"/>
+        <PAGEOBJECT OwnPage="0" PTYPE="4" XPOS="424" YPOS="380" WIDTH="252" HEIGHT="350" RADRECT="0" FRTYPE="0" CLIPEDIT="0" PWIDTH="1" PCOLOR="None" PCOLOR2="None" TXTFILLSH="90" COLUMNS="1" COLGAP="0" NAMEDLST="" SHADE="100" SHADE2="100" GRTYP="0" ROT="0" PLINEART="1" PLINEEND="0" PLINEJOIN="0" LINESP="13.2" LOCALSCX="1" LOCALSCY="1" LOCALX="0" LOCALY="0" PICART="1" PLTSHOW="0" BASEOF="0" textPathType="0" textPathFlipped="0" FLIPPEDH="0" FLIPPEDV="0" IFONT="Adobe Caslon Pro Regular" ISIZE="11" SCALETYPE="1" RATIO="1" PRINTABLE="1" ANNOTATION="0" ANNAME="" TEXTFLOWMODE="0" TEXTFLOW="0" TEXTFLOW2="0" TEXTFLOW3="0" AUTOTEXT="0" EXTRA="0" TEXTRA="0" BEXTRA="0" REXTRA="0" FLOP="0" PFILE="" PFILE2="" PFILE3="" PRFILE="" EPROF="" IRENDER="1" EMBEDDED="1" LOCK="0" LOCKR="0" REVERS="0" TransValue="0" TransValueS="0" TransBlend="0" TransBlendS="0" isTableItem="0" TopLine="0" LeftLine="0" RightLine="0" BottomLine="0" isGroupControl="0" NUMDASH="0" DASHS="" DASHOFF="0" NUMPO="16" POCOOR="0 0 0 0 252 0 252 0 252 0 252 0 252 350 252 350 252 350 252 350 0 350 0 350 0 350 0 350 0 0 0 0 " NUMCO="16" COCOOR="0 0 0 0 252 0 252 0 252 0 252 0 252 350 252 350 252 350 252 350 0 350 0 350 0 350 0 350 0 0 0 0 " NUMGROUP="0" GROUPS="" startArrowIndex="0" endArrowIndex="0" OnMasterPage="" ImageClip="" ImageRes="1" Pagenumber="0" isInline="0" fillRule="1" doOverprint="0" gXpos="0" gYpos="0" gWidth="0" gHeight="0" LAYER="0" BOOKMARK="0" NEXTITEM="-1" BACKITEM="-1">
+            <ITEXT FONT="Adobe Caslon Pro Regular" FONTSIZE="11" FSHADE="90" CH="La primera parada cuando hablamos de la participación en los equipos, son los equipos de trabajo de la comunidad que se enfocan en organizar todos los demás equipos. Ellos te dirán si tu idea ya la desarrollaron y "/>
+            <para/>
+            <ITEXT FONT="Adobe Caslon Pro Regular" FONTSIZE="11" FSHADE="90" CH="con quienes hablar para llevarla a otro nivel. O si tu idea es completamente nueva, te conectarán con algunos recursos para facilitar que se haga una realidad."/>
+            <para/>
+            <para/>
+            <ITEXT FONT="Adobe Caslon Pro Regular" FONTSIZE="11" FSHADE="90" CH="Los desarrolladores núcleo (Core) de Monero nos han dado una cantidad de herramientas que nos permitan el éxito en los equipos de trabajo, incluyendo Mattermost para comunicación y Taiga para la planeación y administración de flujos de tareas para los proyectos. "/>
```

Comment:
"**Los desarrolladores centrales** de Monero...". I've used that meaning in the es.yml file and for consistency it should be the same.

Reply:
"... nos han dado un puñado de herramientas que ayudan a tener equipos de trabajo exitosos, ...". Again, this could be more structured like the suggestion.